### ISSUE TITLE
Missing Linux icon in `Add Game` dialog

### ISSUE THREAD

**Issue by arielj** [Heroic-Games-Launcher/HeroicGamesLauncher#3111]
### Describe the bug

When using the dialog to add custom apps, when `Linux` is selected, there's an error in the console and no icon is displayed.

### Add logs

```text
Could not find icon null 
    at http://localhost:5173/node_modules/.vite/deps/@fortawesome_react-fontawesome.js?v=b6408745:2796:24
    at span
    at div
    at div
    at div
    at DialogContent (http://localhost:5173/src/frontend/components/UI/Dialog/components/DialogContent.tsx:11:33)
    at SideloadDialog (http://localhost:5173/src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.tsx:24:42)
    at dialog
    at div
    at Dialog (http://localhost:5173/src/frontend/components/UI/Dialog/components/Dialog.tsx:14:26)
    at div
    at InstallModal (http://localhost:5173/src/frontend/screens/Library/components/InstallModal/index.tsx:22:82)
    at Library (http://localhost:5173/src/frontend/screens/Library/index.tsx:27:316)
    at RenderedRoute (http://localhost:5173/node_modules/.vite/deps/chunk-3CRLEQ57.js?v=b6408745:3031:5)
    at Routes (http://localhost:5173/node_modules/.vite/deps/chunk-3CRLEQ57.js?v=b6408745:3394:5)
    at main
    at Router (http://localhost:5173/node_modules/.vite/deps/chunk-3CRLEQ57.js?v=b6408745:3341:15)
    at HashRouter (http://localhost:5173/node_modules/.vite/deps/react-router-dom.js?v=b6408745:281:5)
    at div
    at App (http://localhost:5173/src/frontend/App.tsx:31:46)
    at Suspense
    at I18nextProvider (http://localhost:5173/node_modules/.vite/deps/react-i18next.js?v=b6408745:1069:19)
    at GlobalState (http://localhost:5173/src/frontend/state/GlobalState.tsx:256:9)
    at I18nextWithTranslation (http://localhost:5173/node_modules/.vite/deps/react-i18next.js?v=b6408745:1028:31)
o
```

### Steps to reproduce

1. Go to Library
2. Click on "Add Game"
3. In "Select Platform Version to Install" select "Linux"
4. See the error in the devtools console

### Expected behavior

No error in the console.
The penguin/linux icon under the image at the right, like it is for other platforms.

### Heroic Version

Latest Stable

### System Information

Not relevant

**Comment by SarthakD15** [comment]
I want to work on this issue. Can you please assign it to me ?

**Comment by SarthakD15** [comment]
Hey @arielj @Nocccer, I have solved the issue and made a PR #3117 :)
Please review the PR and suggest changes if any !
Thanks !!
